cartridge-cli.lua

```diff
@@ -1222,6 +1154,7 @@ local function form_distribution_dir(source_dir, dest_dir)
     end
     local git = which('git')
     if git ~= nil and fio.path.exists(fio.pathjoin(dest_dir, '.git')) then
+        info('Running `git clean`')
```

Comment:
Let's call "git describe --tag" before and save it to special env variable (e.g. GIT_APPLICATION_VERSION)

Reply:
It could be useful for some users

Reply:
I like your idea, maybe we should write it in the VERSION file.
But in fact, if application version isn't passed explicitly, it will be detected from `git describe --tag`.

Anyway, I would like to discuss it in separated issue.

Reply:
https://github.com/tarantool/cartridge-cli/issues/100